Uchu.World/Handlers/WorldInitializationHandler.cs

```diff
@@ -212,13 +217,14 @@ private async Task SendCharacterXmlDataToClient(Character character, IRakConnect
         /// The generated XML data is based on https://docs.google.com/document/d/1XDh_HcXMjSdaGeniG1dND5CA7jOFXIPA_fxCnjvjaO4/edit#
         /// </remarks>
         /// <param name="character">The character to generate the XML data for</param>
+        /// <param name="GMLevel">The GM level of the user</param>
         /// <returns>XmlData conform with the LU Char Data XML Format</returns>
-        private static XmlData GenerateCharacterXmlData(Character character)
+        private static XmlData GenerateCharacterXmlData(Character character, int GMLevel)
```

Comment:
Use lowercase parameter names (e.g. gmLevel)

Reply:
Done

Reply:
@Jettford the `param` in the documentation/comment still has GMLevel instead of gmLevel (line 220 and 301)